loadouts/includes/classes-list.sqf

```diff
@@ -18,6 +18,8 @@ _isReconLeader = (_type == "reconleader")||(_type == "Recon Team Leader");
 _isRecon = (_type == "recon")||(_type == "Recon Scout");
 _isReconDemo = (_type == "recondemo")||(_type == "Recon Demo Specialist");
 _isReconMedic = (_type == "reconmedic")||(_type == "Recon Paramedic");
+_isReconAT = (_type == "reconat")||(_type == "Recon Scout (AT)");
+_isReconMarksman = (_type == "reconmarksman")||(_type == "Recon Marksman");
```

Comment:
All of the structures would also need to be changed to allow these. Should also have been a separate PR.